src/components/saas-list/saas-list.html

```diff
@@ -3,8 +3,13 @@ <h1 ng-if="$ctrl.saasTitle">{{$ctrl.saasTitle}}</h1>
   <div class="saas-list" items="$ctrl.saasOfferings">
     <div class="card" ng-repeat="item in $ctrl.saasOfferings">
       <a href="" class="card-content" ng-click="$ctrl.handleClick(item)">
-        <span class="card-icon {{item.icon}}"><img ng-if="item.image" ng-src="{{item.image}}"></span>
-        <span class="card-title">{{item.title}}</span>
+        <div ng-if="!item.image" class="card-icon">
+          <span class="icon {{item.icon}}"></span>
+        </div>
+        <div ng-if="item.image" class="card-icon">
+          <img ng-src="{{item.image}}" alt="">
+        </div>
```

Comment:
Since you have the `card-icon` div either way, I suggest

```html
<div class="card-icon">
  <img ng-if="item.image" ng-src="{{item.image}}" alt="">
  <span ng-if="!item.image" class="icon {{item.icon}}" aria-hidden="true"></span>
</div>
```

We should put `aria-hidden="true"` as well on the font icon while in the code.